### DISCUSSION TITLE
评论板块等的静态资源问题想要问问

### DISCUSSION THREAD

**Discussion by Bigwind110** [EveSunMaple/Frosti#23] [Q&A] [answered]
评论板块的rection图片和css等静态资源是否能够使用自己的cdn或图床提升速度，改变是否会影响评论的正常功能？

**Comment by EveSunMaple** [comment, marked as answer]
可以，不会影响正常使用。

**Comment by Bigwind110** [comment]
谢谢

**Comment by Bigwind110** [comment]
后续的版本会考虑点击博客栏目，直接跳转到所有博客并展开侧边栏吗？考虑到可能用户只想点击一次
还有就是你的博客https://www.saroprock.com/blog/categories 报错了
![image](https://github.com/user-attachments/assets/3efb7305-965d-4beb-aaee-2262239222d6)

**Reply by EveSunMaple** [reply]
这个是我自己博客的问题，没有删掉这个导航而已
